### PR TITLE
[Merged by Bors] - Cascaded Shadow Maps, EnvironmentMap Lighting, Tonemapping, and Color Grading

### PR DESCRIPTION
**Pull request by cart**

### CODE REVIEW

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 1]
```diff
@@ -416,6 +416,102 @@ The Bevy ECS team has worked closely with `@jakobhellerman`, the author of [`bev
 
 It's a great tool that we are looking to build on to create a first party solution: you should strongly consider adding it to your toolbox.
 
+## Cascaded Shadow Maps
+
+<div class="release-feature-authors">authors: @danchia, @superdump</div>
```
Comment (superdump) [inline content/news/2023-03-04-bevy-0.10/index.md:421]:
```suggestion
<div class="release-feature-authors">authors: @danchia, Rob Swain (@superdump)</div>
```

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 2]
```diff
@@ -416,6 +416,102 @@ The Bevy ECS team has worked closely with `@jakobhellerman`, the author of [`bev
...
+
+Bevy uses "shadow maps" to cast shadows for lights / objects. Previous versions of Bevy used a simple but limited shadow map implementation. For a given light, you would define the resolution of the shadow map _and_ a manual "view projection" that would determine how the shadow is cast. This had a number of downsides:
```
Comment (superdump) [inline content/news/2023-03-04-bevy-0.10/index.md:423]:
```suggestion
Bevy uses "shadow maps" to cast shadows for lights / objects. Previous versions of Bevy used a simple but limited shadow map implementation for directional light sources. For a given light, you would define the resolution of the shadow map _and_ a manual "view projection" that would determine how the shadow is cast. This had a number of downsides:
```

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 3]
```diff
@@ -416,6 +416,102 @@ The Bevy ECS team has worked closely with `@jakobhellerman`, the author of [`bev
...
+
+And this is what the PBR material looks like with environment map lighting:
+
+![env map after](env_map_after.png)
+
+For scenes that need constant lighting (especially outdoor scenes), environment maps are a great solution. And because environment maps are arbitrary images, artists have a lot of control over the character of the scene's lighting.
+
+## More Tonemapping Choices
+
+<div class="release-feature-authors">authors: @JMS55</div>
```
Comment (superdump) [inline content/news/2023-03-04-bevy-0.10/index.md:455]:
```suggestion
<div class="release-feature-authors">authors: @DGriffin91 @JMS55</div>
```

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 4]
```diff
@@ -416,6 +416,102 @@ The Bevy ECS team has worked closely with `@jakobhellerman`, the author of [`bev
...
+
+Tonemapping is the process of transforming raw HDR information into actual "screen colors". In previous versions of Bevy you had exactly two tonemapping options: Reinhard or disabled tonemapping. In **Bevy 0.10** we've added a ton of choices!
```
Comment (superdump) [inline content/news/2023-03-04-bevy-0.10/index.md:457]:
```suggestion
Tonemapping is the process of transforming raw HDR information into actual "screen colors". In previous versions of Bevy you had exactly two tonemapping options: Reinhard Luminance or disabled tonemapping. In **Bevy 0.10** we've added a ton of choices!
```

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 5]
```diff
@@ -416,6 +416,102 @@ The Bevy ECS team has worked closely with `@jakobhellerman`, the author of [`bev
...
+
+![reinhard](tm_reinhard.png)
+
+### Reinhard Luminance
+
+This is what we had in previous versions of Bevy. It is still our default algorithm.
+
+![reinhard luminance](tm_reinhard_luminance.png)
+
+### ACES
```
Comment (superdump) [inline content/news/2023-03-04-bevy-0.10/index.md:475]:
```suggestion
### ACES Fitted
```

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 6]
```diff
@@ -416,6 +416,102 @@ The Bevy ECS team has worked closely with `@jakobhellerman`, the author of [`bev
...
+
+### Blender Filmic
+
+From the 3D software package we know and love!
+
+![Blender Filmic](tm_blender_filmic.png)
+
+## Color Grading Control
+
+<div class="release-feature-authors">authors: @JMS55</div>
```
Comment (superdump) [inline content/news/2023-03-04-bevy-0.10/index.md:501]:
```suggestion
<div class="release-feature-authors">authors: @DGriffin91</div>
```

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 7]
```diff
@@ -416,6 +416,102 @@ The Bevy ECS team has worked closely with `@jakobhellerman`, the author of [`bev
...
+
+We've added some basic control over color grading parameters such as exposure, gamma, "pre tonemapping saturation", and "post tone mapping saturation". These can be configured per-camera using the new [`ColorGrading`] component
```
Comment (superdump) [inline content/news/2023-03-04-bevy-0.10/index.md:503]:
```suggestion
We've added some basic control over color grading parameters such as exposure, gamma, "pre-tonemapping saturation", and "post-tonemapping saturation". These can be configured per-camera using the new [`ColorGrading`] component
```

**Review by JMS55 — APPROVED** [review]
Lgtm! Maybe add a screenshot somewhere of the tonemapping example without the controls hidden.

**Comment by cart** [comment]
Given how many images we already have for tonemapping, idk if including another image is the right call. The controls on their own don't really add new information about tonemapping, so the image would only serve to show that the controls exist.

**Pull request closed without merging**